saw-core/src/SAWCore/SharedTerm.hs

```diff
@@ -3100,49 +3059,41 @@ scCloseTerm close sc ec body = do
 -- | Deconstruct a lambda term into a bound variable and a body, using
 -- a fresh 'ExtCns' for the bound variable.
 scAsLambda :: SharedContext -> Term -> IO (Maybe (ExtCns Term, Term))
-scAsLambda sc t =
+scAsLambda _sc t =
```

Comment:
Some of the functions in this module (e.g., `scAsLambda`) no longer use their `SharedContext` argument, and they could conceivably be made pure instead of returning `IO`. Do you think it would be worth doing so, or would you prefer to keep these for consistency with the types of other `sc*`-prefixed functions?

Reply:
The `scAs*` functions were useful during the transition period when I was converting code to use the named-variable API functions, but terms still used de Bruijn indices internally. Now we can just remove them and use pure functions from the Recognizer library instead.